python/py-line_profiler/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-line_profiler
+version             4.1.2
+revision            0
+supported_archs     noarch
+platforms           {darwin any}
```

Comment:
this doesn't appear to be "noarch" nor "{darwin any}". Please verify these things before submitting a PR.

Reply:
Thank you. Fixed.